src/DependencyInjection/Configuration.php

```diff
@@ -61,6 +61,7 @@ private function createClientsNode()
                         ->prototype('scalar')->end()
                     ->end()
                     ->arrayNode('request_options')
+                        ->info('Details: http://docs.guzzlephp.org/en/stable/request-options.html')
```

Comment:
I would either use http://docs.guzzlephp.org/en/latest/request-options.html here or change the link in the README.md so we use the same links when linking for help.